config/initializers/008-rack-cors.rb

```diff
@@ -34,6 +34,8 @@ def self.apply_headers(cors_origins, env, headers)
     origin = nil
 
     if cors_origins
+      cors_origins = cors_origins.map { |x| x.chomp('/') }
+
```

Comment:
should we not chomp it a bit earlier at ` cors_origins += SiteSetting.cors_origins.split('|') if SiteSetting.cors_origins.present?`

Reply:
I guess if we keep the global origins pre-chomped ... maybe all of this extra work can be just totally avoided by adding a validation on the site setting?

Reply:
The site setting validation is now added.